htdocs/langs/en_US/companies.lang

```diff
@@ -381,11 +381,10 @@ ProspectsByStatus=Prospects by status
 NoParentCompany=None
 ExportCardToFormat=Export card to format
 ContactNotLinkedToCompany=Contact not linked to any third party
-DolibarrLogin=Dolibarr login
-NoDolibarrAccess=No Dolibarr access
-ExportDataset_company_1=Third-parties (organizations/natural persons) and attributes
-ExportDataset_company_2=Third-parties additional contacts/addresses and attributes
-ExportDataset_company_3=Third-parties payment modes (bank accounts)
+DolibarrLogin=External user login
```

Comment:
The key "Dolibarr Login" is a generic labelfor any login to the application (for external user but also internal users).